src/rasmus_fuel/realistic.py

```diff
@@ -0,0 +1,198 @@
+import numpy as np
+import numpy.typing as npt
+
+
+def power_maintain_sog(
```

Comment:
I'd do the following

```python
PHYSICS_PARAM = {
    "air_mass_density": 1.225,
    "wind_resistance_coefficient" 0.4,
    # ... etc
}

EXAMPLE_VESSEL_PARAM = {
    "waterline_width": 30,
    "waterline_length": 210,
    # ... etc.
}

def power_maintain_sog(
    # ... etc.
    physics_param: dict = None,
    vessel_param: dict = None,
    **kwargs,
) -> npt.ArrayLike:
    # ...
    if physics_param is None:
        physics_param = PHYSICS_PARAM
    if vessel_param is None:
        vessel_param = EXAMPLE_VESSEL_PARAM
    # ...
```

```

Reply:
As the parameters are not too many anyway

https://github.com/willirath/rasmus_fuel/blob/db8cae42a0d7c615731ca465fc5a156d1c9e6978/src/rasmus_fuel/realistic.py#L14_L26

we could also just drop the containing dictionary and have them as explicit keyword arguments.